developers/wallets/native/index.md

```diff
@@ -94,7 +94,12 @@ Once done, your Wallet should now be enabled to handle various credential relate
 
 Now that your native wallet has been registered with CHAPI, it can receive requests to store credentials at the URL stated in the `credential_handler.url` of the `manifest.json` above.
 
-The endpoint declared there (`/switchboard` in the example above) will receive a `request` query parameter that will include a URL encoded object with the following properties:
+CHAPI will use the endpoint declared (`/switchboard` in the example above) when it provides links to available wallets in the user's browser:
+
+![Choose a wallet modal presenting all preregisterd wallet systems which can be clicked on to proceed to storing the credentials there.](/images/VeresCHAPIaccept.png)
```

Comment:
```suggestion
![Choose a wallet modal presenting all preregistered wallet systems which can be clicked on to proceed to store the credentials there.](/images/VeresCHAPIaccept.png)
```

Reply:
ah realize this is just the alt text but still worth fixing up

Reply:
Alt text is still valuable text. ❤️